app.json

```diff
@@ -1,4 +1,4 @@
 {
-  "name": "Zingo!",
+  "name": "ZingoMobile",
```

Comment:
upsss, did you need that this `name` is `ZingoMobile`???? for some reason...

Reply:
![Screenshot from 2023-02-03 16-25-17](https://user-images.githubusercontent.com/111296437/216655139-b59346f7-5c3a-4dfe-9fff-7f6a7f234681.png)

that fixed this bug in yarn react-native start

Reply:
this is strange... this is my console... (i have no problems with `Zingo!`...)
![image](https://user-images.githubusercontent.com/43755989/216709084-4de9312e-24ed-4fd9-a806-969f65e536ed.png)